src/routes/(home)/+layout.svelte

```diff
@@ -0,0 +1,13 @@
+<script>
```

Comment:
Added a "group" here so that home would have the scrolling stuff, but other (new) pages would not